go_hound/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM socrata/golang
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y install rsync nodejs npm && \
+    npm -g install react-tools && \
+    ln -nsf /usr/bin/nodejs /usr/bin/node
+
+RUN go get github.com/etsy/hound/cmds/...
```

Comment:
Isn't there an install step that is needed? The hound Dockerfile from etsy has the following line:
```
RUN go-wrapper install github.com/etsy/hound/cmds/houndd
```